autoapi/documenters.py

```diff
@@ -50,6 +62,9 @@ def import_object(self):
                 self._method_parent = parent
                 return True
 
+        # If we get here, the object was not found. Emit a warning as autodoc does.
+        LOGGER.warning("Failed to import %s '%s' [autoapi.import]", self.directivetype, self.fullname, type="autoapi", subtype="import")
```

Comment:
I believe that the inclusion of `[autoapi.import]` at the end of the message isn't required because the message type is output by default.